store/src/java/com/zimbra/cs/mailbox/ContactBackupThread.java

```diff
@@ -16,9 +16,12 @@
  */
```

Comment:
if (shutdownRequested) {
                ZimbraLog.contactbackup.info("shutting down thread.");
                shutdown();
                return;
}

This block can be outside the for loop

Reply:
just wanted to make sure that, if shutdown is requested at any point of time, thread should not process further mailboxes. So kept in the loop on mailboxes.